requirements.pinned.txt

```diff
@@ -24,3 +24,5 @@
 
 # 2.13.5 break Invenio-OAIServer date granularity validation.
 marshmallow==2.13.4
+# 0.14 have changed CSRF validation.
+Flask-WTF==0.13.1
```

Comment:
Issues with 0.14 should have been fixed in upstream Invenio, but not full sure so I leave it pinned.